src/ts/localization/de.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Can we move the `*.json` files into an additional subfolder called `languages` or something?